src/main/java/com/emc/object/s3/S3Config.java

```diff
@@ -331,6 +348,11 @@ public S3Config withUseV2Signer(boolean useV2Signer) {
         return this;
     }
 
+    public S3Config withMaxConnectionIdleTime(int maxConnectionIdleTime) {
+        setMaxConnectionIdleTime(maxConnectionIdleTime);
+        return this;
+    }
+
     @Override
     public String toString() {
         return "S3Config{" +
```

Comment:
Shall it be added to `toString()` as well?

Reply:
I just checked ConnectTimeout and do the same way, to add the methods in ObjectConfig(not in S3Config) and no changes in toString.

Reply:
If you do code blame, you'll see what I meant. It looks like Haoyuan missed the change, and wasn't spotted by code review. Similar code place is `S3Config.toString()`, all members were added. Additional information will be useful for the troubleshooting especially now it's a configurable variable.